src/v2/Apps/Consign/Routes/MarketingLanding/Components/FAQ.tsx

```diff
@@ -1,119 +1,155 @@
 import { Expandable, Join, Spacer, Text } from "@artsy/palette"
+import { AnalyticsSchema, useTracking } from "v2/System"
+import { ContextModule, OwnerType } from "@artsy/cohesion"
 
 export const FAQ: React.FC = () => {
+  const { trackEvent } = useTracking()
+
+  const trackClickedFAQ = (label: string) => {
+    trackEvent({
+      action_type: AnalyticsSchema.ActionType.ClickedFAQ,
+      context_module: ContextModule.consignSubmissionFlow,
+      context_owner_type: OwnerType.consignmentSubmission,
+      subject: label,
+    })
```

Comment:
2 pages use the component: /consign and /consign/submission/:id/thank-you, should we track the event for both of them?

Reply:
thanks for this comment 🙏  I clarified with data team: we are tracking a FAQ click only after a submission is made now.